test/puppetlabs/trapperkeeper/testutils/logging_test.clj

```diff
@@ -0,0 +1,150 @@
+(ns puppetlabs.trapperkeeper.testutils.logging-test
+  (:require
+   [clojure.test :refer :all]
+   [clojure.tools.logging :as log]
+   [puppetlabs.kitchensink.core :as kitchensink]
+   [puppetlabs.trapperkeeper.logging :refer [reset-logging root-logger-name]]
+   [puppetlabs.trapperkeeper.testutils.logging :as tgt :refer [event->map]])
+  (:import
+   [org.slf4j Logger LoggerFactory]))
+
+;; Without this, "lein test NAMESPACE" and :only invocations may fail.
+(use-fixtures :once (fn [f] (reset-logging) (f)))
```

Comment:
I'd assumed it was because in the `lein test' case, some other namespace (that runs first) was initializing logging, but for something like `lein test puppetlabs.trapperkeeper.testutils.logging-test`, we need to do it manually (via the fixture).